### ISSUE TITLE
Не работает кнопка выхода из пользователя

### ISSUE THREAD

**Issue by RemizovDmitry** [RemizovDmitry/qa#2]
**Шаги для воспроизведения:**

1. В главном окне приложения нажать кнопку выхода из учетной записи пользователя(см. аттач 1)

**Результат:**
Ничего не происходит
**ОР:**

Выйти на окно регистрации\входа в систему
![image](https://user-images.githubusercontent.com/45583813/49460825-55a0ec00-f803-11e8-9863-9e991e6a2b07.png)

**Comment by hardbreather** [comment]
Да. пока не делали, так как не является критическим или блокирующим фактором.
